include/ttmlir/Dialect/TTKernel/IR/TTKernelOps.td

```diff
@@ -96,6 +96,15 @@ def TTKernel_AddOp : TTKernel_Op<"add"> {
     let arguments = (ins I32:$dst_index);
 }
 
+def TTKernel_DivOp : TTKernel_Op<"div"> {
+    let summary = "Divide operation";
+    let description = [{
+      Divide operation
+    }];
+
+    let arguments = (ins I32:$dst_index);
```

Comment:
We shouldn't need to update TTKernel dialect, the AddOp above even is incorrect.  We are still bringing up this dialect.  The rest looks good though!

Reply:
Sure, i will remove addop to. Other elwise ops are supported?

Reply:
Leave the add op, I think it's being used by a test, but we're going to rework it.